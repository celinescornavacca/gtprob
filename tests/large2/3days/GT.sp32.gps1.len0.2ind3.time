0.38user 0.20system 0:05.39elapsed 10%CPU (0avgtext+0avgdata 44172maxresident)k
389167inputs+0outputs (79major+16490minor)pagefaults 0swaps
