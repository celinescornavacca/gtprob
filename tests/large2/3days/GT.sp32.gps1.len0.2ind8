(((RA0,SA0),((TA0,(UA0,VA0)),((NA0,OA0),AA0))),((((HA0,((IA0,(PA0,QA0)),(CB0,DB0))),((AB0,BB0),LA0)),(((EA0,FA0),BA0),((EB0,FB0),(JA0,KA0)))),((((WA0,XA0),GA0),(MA0,(YA0,ZA0))),(CA0,DA0))));
