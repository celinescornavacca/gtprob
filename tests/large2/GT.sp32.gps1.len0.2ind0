((((CA0,(EA0,(((MA0,NA0),(AB0,BB0)),GA0))),(FA0,(CB0,DB0))),((JA0,(YA0,ZA0)),(KA0,LA0))),((AA0,BA0),(((WA0,XA0),(OA0,PA0)),(((DA0,(QA0,RA0)),((SA0,TA0),((HA0,IA0),(EB0,FB0)))),(UA0,VA0)))));
