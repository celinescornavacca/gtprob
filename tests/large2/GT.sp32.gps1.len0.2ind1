(((SA0,TA0),(DA0,(((((OA0,(AB0,BB0)),LA0),(PA0,QA0)),(YA0,ZA0)),(KA0,(MA0,NA0))))),((AA0,BA0),(((FA0,GA0),EA0),((((((UA0,VA0),RA0),(WA0,XA0)),(HA0,(IA0,JA0))),((EB0,FB0),(CB0,DB0))),CA0))));
