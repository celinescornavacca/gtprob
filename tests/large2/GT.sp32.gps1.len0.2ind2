((CA0,(DA0,(EA0,((OA0,(PA0,QA0)),((TA0,UA0),IA0))))),((AA0,(HA0,((RA0,SA0),(LA0,(((AB0,BB0),WA0),(EB0,FB0)))))),(((BA0,((JA0,KA0),GA0)),(XA0,YA0)),(((((CB0,DB0),ZA0),VA0),FA0),(MA0,NA0)))));
