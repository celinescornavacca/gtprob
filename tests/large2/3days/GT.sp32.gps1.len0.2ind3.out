-41.3854323451
